manifests/postgres_exporter.pp

```diff
@@ -0,0 +1,209 @@
+# Class: prometheus::postgres_exporter
+#
+# This module manages prometheus node postgres_exporter
+#
+# Parameters:
+#  [*arch*]
+#  Architecture (amd64 or i386)
+#
+#  [*bin_dir*]
+#  Directory where binaries are located
+#
+#  [*addr*]
+#  Array of address of one or more redis nodes. Defaults to redis://localhost:6379
+#
+#  [*download_extension*]
+#  Extension for the release binary archive
+#
+#  [*download_url*]
+#  Complete URL corresponding to the where the release binary archive can be downloaded
+#
+#  [*download_url_base*]
+#  Base URL for the binary archive
+#
+#  [*extra_groups*]
+#  Extra groups to add the binary user to
+#
+#  [*extra_options*]
+#  Extra options added to the startup command
+#  For a full list of the exporter's supported extra options
+#  please refer to https://github.com/wrouesnel/postgres_exporter
+#
+#  [*group*]
+#  Group under which the binary is running
+#
+#  [*init_style*]
+#  Service startup scripts style (e.g. rc, upstart or systemd)
+#
+#  [*install_method*]
+#  Installation method: url or package (only url is supported currently)
```

Comment:
`only url is supported currently`.  Is that true?

Reply:
I'm not aware of a packaged version of postgres_exporter, if it exsist, its not tested.